pre-commit.patch

```diff
@@ -0,0 +1,141 @@
+diff --git a/src/preferences/dialog/dlgprefdeck.h b/src/preferences/dialog/dlgprefdeck.h
```

Comment:
This file needs to be removed.

Reply:
Done ! It got added to a commit by mistake, good catch, thanks!